.github/workflows/main.yml

```diff
@@ -5,11 +5,25 @@ env:
   TOOLCHAIN_TAG: 0.16.0
 jobs:
   tools:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest-8-cores
     steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: recursive
+      - name: Install libarchive-tools
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends build-essential autoconf automake libarchive-dev libarchive-tools
+
+      - name: Download, build and install xgenext2fs
+        run: |
```

Comment:
Because `.deb` from genext2fs repo builds for `debian` and not `ubuntu`?
If so, I think this deserves a comment.